feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -286,10 +286,10 @@ func TestEstablishAndDisconnect(t *testing.T) {
 	// Send Cease Notification from ATE to DUT
 	t.Log("Send Cease Notification from ATE to DUT")
 	ate.Actions().NewBGPPeerNotification().WithCode(6).WithSubCode(6).WithPeers(bgpPeer).Send(t)
-
+	time.Sleep(time.Second * 3)
```

Comment:
Is this sleep still needed?

Reply:
Yeah, Since we are sending notification from Ixia , 2-3 secs is needed.